gammapy/image/tests/test_mask.py

```diff
@@ -16,7 +16,21 @@ def test_random_creation():
     assert excluded[0].size != 0
 
 
-@pytest.mark.xfail
+@requires_dependency('scipy')
 def test_distance_image():
-    pass
-    # TODO
+    mask = ExclusionMask.empty(nxpix=3, nypix=2)
+    distance = mask.distance_image.data
+    expected = [[-1., -1.41421356, -2.23606798],
+                [-2., -2.23606798, -2.82842712]]
+    assert_allclose(distance, expected)
+
+    mask = ExclusionMask.empty(nxpix=3, nypix=2, fill=1.)
+    distance = mask.distance_image.data
+    expected = [[1e10, 1e10, 1e10], [1e10, 1e10, 1e10]]
```

Comment:
For constant arrays, there's no need to duplicate the values, you can just compare array to scalar and Numpy broadcasting will take care of comparing every value:
```python
assert_allclose(distance, 1e10)
```